manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: freifunkMUC.FreifunkMeet
+PackageVersion: 2024.3.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
```

Comment:
should also be align to: https://github.com/microsoft/winget-pkgs/blob/master/manifests/j/Jitsi/Meet/2024.3.0/Jitsi.Meet.installer.yaml#L12